### PR TITLE
Update `dask`/`distributed` pinnings to exclude nightlies

### PR DESCRIPTION
**Pull request by charlesbluca**
Adjusts the pinnings of `dask` and `distributed` packages published after Dask nightlies became available to prevent nightly packages from getting installed with stable conda-forge packages; should make the `dask-core` and `distributed` pinnings of these packages consistent with the changes made in https://github.com/conda-forge/dask-feedstock/pull/191 and https://github.com/conda-forge/distributed-feedstock/pull/218.

cc @jakirkham @jrbourbeau 

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by charlesbluca** [comment]
Output of `show_diff.py`:

<details>

```
noarch::dask-2021.12.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2021.12.0.*",
-    "distributed 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
+    "distributed >=2021.12.0,<2021.12.1.0a0",
noarch::dask-2022.1.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.1.0.*",
-    "distributed 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
+    "distributed >=2022.1.0,<2022.1.1.0a0",
noarch::dask-2022.1.1-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.1.1.*",
-    "distributed 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
+    "distributed >=2022.1.1,<2022.1.2.0a0",
noarch::dask-2022.2.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.2.0.*",
-    "distributed 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
+    "distributed >=2022.2.0,<2022.2.1.0a0",
noarch::dask-2022.2.1-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.2.1.*",
-    "distributed 2022.2.1.*",
+    "dask-core >=2022.2.1,<2022.2.2.0a0",
+    "distributed >=2022.2.1,<2022.2.2.0a0",
noarch::dask-2022.3.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.3.0.*",
-    "distributed 2022.3.0.*",
+    "dask-core >=2022.3.0,<2022.3.1.0a0",
+    "distributed >=2022.3.0,<2022.3.1.0a0",
noarch::dask-2022.3.0-pyhd8ed1ab_1.tar.bz2
-    "dask-core 2022.3.0.*",
-    "distributed 2022.3.0.*",
+    "dask-core >=2022.3.0,<2022.3.1.0a0",
+    "distributed >=2022.3.0,<2022.3.1.0a0",
noarch::dask-2022.4.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.4.0.*",
-    "distributed 2022.4.0.*",
+    "dask-core >=2022.4.0,<2022.4.1.0a0",
+    "distributed >=2022.4.0,<2022.4.1.0a0",
noarch::dask-2022.4.1-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.4.1.*",
-    "distributed 2022.4.1.*",
+    "dask-core >=2022.4.1,<2022.4.2.0a0",
+    "distributed >=2022.4.1,<2022.4.2.0a0",
noarch::dask-2022.4.2-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.4.2.*",
-    "distributed 2022.4.2.*",
+    "dask-core >=2022.4.2,<2022.4.3.0a0",
+    "distributed >=2022.4.2,<2022.4.3.0a0",
noarch::dask-2022.5.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.5.0.*",
-    "distributed 2022.5.0.*",
+    "dask-core >=2022.5.0,<2022.5.1.0a0",
+    "distributed >=2022.5.0,<2022.5.1.0a0",
noarch::dask-2022.5.1-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.5.1.*",
-    "distributed 2022.5.1.*",
+    "dask-core >=2022.5.1,<2022.5.2.0a0",
+    "distributed >=2022.5.1,<2022.5.2.0a0",
noarch::dask-2022.5.2-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.5.2.*",
-    "distributed 2022.5.2.*",
+    "dask-core >=2022.5.2,<2022.5.3.0a0",
+    "distributed >=2022.5.2,<2022.5.3.0a0",
noarch::dask-2022.6.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.6.0.*",
-    "distributed 2022.6.0.*",
+    "dask-core >=2022.6.0,<2022.6.1.0a0",
+    "distributed >=2022.6.0,<2022.6.1.0a0",
noarch::dask-2022.6.1-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.6.1.*",
-    "distributed 2022.6.1.*",
+    "dask-core >=2022.6.1,<2022.6.2.0a0",
+    "distributed >=2022.6.1,<2022.6.2.0a0",
noarch::dask-2022.7.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.7.0.*",
-    "distributed 2022.7.0.*",
+    "dask-core >=2022.7.0,<2022.7.1.0a0",
+    "distributed >=2022.7.0,<2022.7.1.0a0",
noarch::dask-2022.7.1-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.7.1.*",
-    "distributed 2022.7.1.*",
+    "dask-core >=2022.7.1,<2022.7.2.0a0",
+    "distributed >=2022.7.1,<2022.7.2.0a0",
noarch::dask-2022.8.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.8.0.*",
-    "distributed 2022.8.0.*",
+    "dask-core >=2022.8.0,<2022.8.1.0a0",
+    "distributed >=2022.8.0,<2022.8.1.0a0",
noarch::dask-2022.8.0-pyhd8ed1ab_1.tar.bz2
-    "dask-core ==2022.8.0,!=2022.8.0a*.*",
-    "distributed ==2022.8.0,!=2022.8.0a*.*",
+    "dask-core >=2022.8.0,<2022.8.1.0a0",
+    "distributed >=2022.8.0,<2022.8.1.0a0",
noarch::distributed-2022.2.1-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.2.1.*",
+    "dask-core >=2022.2.1,<2022.2.2.0a0",
noarch::distributed-2022.3.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.3.0.*",
+    "dask-core >=2022.3.0,<2022.3.1.0a0",
noarch::distributed-2022.4.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.4.0.*",
+    "dask-core >=2022.4.0,<2022.4.1.0a0",
noarch::distributed-2022.4.1-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.4.1.*",
+    "dask-core >=2022.4.1,<2022.4.2.0a0",
noarch::distributed-2022.4.2-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.4.2.*",
+    "dask-core >=2022.4.2,<2022.4.3.0a0",
noarch::distributed-2022.5.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.5.0.*",
+    "dask-core >=2022.5.0,<2022.5.1.0a0",
noarch::distributed-2022.5.1-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.5.1.*",
+    "dask-core >=2022.5.1,<2022.5.2.0a0",
noarch::distributed-2022.5.2-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.5.2.*",
+    "dask-core >=2022.5.2,<2022.5.3.0a0",
noarch::distributed-2022.6.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.6.0.*",
+    "dask-core >=2022.6.0,<2022.6.1.0a0",
noarch::distributed-2022.6.1-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.6.1.*",
+    "dask-core >=2022.6.1,<2022.6.2.0a0",
noarch::distributed-2022.6.1-pyhd8ed1ab_1.tar.bz2
-    "dask-core 2022.6.1.*",
+    "dask-core >=2022.6.1,<2022.6.2.0a0",
noarch::distributed-2022.7.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.7.0.*",
+    "dask-core >=2022.7.0,<2022.7.1.0a0",
noarch::distributed-2022.7.1-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.7.1.*",
+    "dask-core >=2022.7.1,<2022.7.2.0a0",
noarch::distributed-2022.8.0-pyhd8ed1ab_0.tar.bz2
-    "dask-core 2022.8.0.*",
+    "dask-core >=2022.8.0,<2022.8.1.0a0",
noarch::distributed-2022.8.0-pyhd8ed1ab_1.tar.bz2
-    "dask-core ==2022.8.0,!=2022.8.0a*.*",
+    "dask-core >=2022.8.0,<2022.8.1.0a0",
linux-64::distributed-2021.12.0-py310hff52083_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-64::distributed-2021.12.0-py37h89c1867_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-64::distributed-2021.12.0-py37h9c2f6ca_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-64::distributed-2021.12.0-py38h578d9bd_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-64::distributed-2021.12.0-py39hf3d152e_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-64::distributed-2022.1.0-py310hff52083_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-64::distributed-2022.1.0-py37h89c1867_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-64::distributed-2022.1.0-py37h9c2f6ca_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-64::distributed-2022.1.0-py38h578d9bd_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-64::distributed-2022.1.0-py39hf3d152e_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-64::distributed-2022.1.1-py310hff52083_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-64::distributed-2022.1.1-py37h89c1867_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-64::distributed-2022.1.1-py37h9c2f6ca_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-64::distributed-2022.1.1-py38h578d9bd_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-64::distributed-2022.1.1-py39hf3d152e_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-64::distributed-2022.2.0-py310hff52083_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-64::distributed-2022.2.0-py37h89c1867_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-64::distributed-2022.2.0-py37h9c2f6ca_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-64::distributed-2022.2.0-py38h578d9bd_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-64::distributed-2022.2.0-py39hf3d152e_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-aarch64::distributed-2021.12.0-py310hbbe02a8_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-aarch64::distributed-2021.12.0-py37h4bcbb9b_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-aarch64::distributed-2021.12.0-py37hd9ded2f_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-aarch64::distributed-2021.12.0-py38h2063c64_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-aarch64::distributed-2021.12.0-py39ha65689a_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-aarch64::distributed-2022.1.0-py310hbbe02a8_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-aarch64::distributed-2022.1.0-py37h4bcbb9b_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-aarch64::distributed-2022.1.0-py37hd9ded2f_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-aarch64::distributed-2022.1.0-py38h2063c64_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-aarch64::distributed-2022.1.0-py39ha65689a_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-aarch64::distributed-2022.1.1-py310hbbe02a8_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-aarch64::distributed-2022.1.1-py37h4bcbb9b_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-aarch64::distributed-2022.1.1-py37hd9ded2f_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-aarch64::distributed-2022.1.1-py38h2063c64_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-aarch64::distributed-2022.1.1-py39ha65689a_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-aarch64::distributed-2022.2.0-py310hbbe02a8_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-aarch64::distributed-2022.2.0-py37h4bcbb9b_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-aarch64::distributed-2022.2.0-py37hd9ded2f_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-aarch64::distributed-2022.2.0-py38h2063c64_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-aarch64::distributed-2022.2.0-py39ha65689a_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-ppc64le::distributed-2021.12.0-py310hd032262_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-ppc64le::distributed-2021.12.0-py37h35e4cab_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-ppc64le::distributed-2021.12.0-py37hadc05a3_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-ppc64le::distributed-2021.12.0-py38hf8b3453_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-ppc64le::distributed-2021.12.0-py39hc1b9086_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
linux-ppc64le::distributed-2022.1.0-py310hd032262_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-ppc64le::distributed-2022.1.0-py37h35e4cab_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-ppc64le::distributed-2022.1.0-py37hadc05a3_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-ppc64le::distributed-2022.1.0-py38hf8b3453_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-ppc64le::distributed-2022.1.0-py39hc1b9086_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
linux-ppc64le::distributed-2022.1.1-py310hd032262_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-ppc64le::distributed-2022.1.1-py37h35e4cab_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-ppc64le::distributed-2022.1.1-py37hadc05a3_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-ppc64le::distributed-2022.1.1-py38hf8b3453_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-ppc64le::distributed-2022.1.1-py39hc1b9086_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
linux-ppc64le::distributed-2022.2.0-py310hd032262_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-ppc64le::distributed-2022.2.0-py37h35e4cab_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-ppc64le::distributed-2022.2.0-py37hadc05a3_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-ppc64le::distributed-2022.2.0-py38hf8b3453_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
linux-ppc64le::distributed-2022.2.0-py39hc1b9086_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
osx-64::distributed-2021.12.0-py310h2ec42d9_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
osx-64::distributed-2021.12.0-py37h5186d4c_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
osx-64::distributed-2021.12.0-py37hf985489_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
osx-64::distributed-2021.12.0-py38h50d1736_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
osx-64::distributed-2021.12.0-py39h6e9494a_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
osx-64::distributed-2022.1.0-py310h2ec42d9_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
osx-64::distributed-2022.1.0-py37h5186d4c_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
osx-64::distributed-2022.1.0-py37hf985489_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
osx-64::distributed-2022.1.0-py38h50d1736_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
osx-64::distributed-2022.1.0-py39h6e9494a_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
osx-64::distributed-2022.1.1-py310h2ec42d9_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
osx-64::distributed-2022.1.1-py37h5186d4c_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
osx-64::distributed-2022.1.1-py37hf985489_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
osx-64::distributed-2022.1.1-py38h50d1736_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
osx-64::distributed-2022.1.1-py39h6e9494a_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
osx-64::distributed-2022.2.0-py310h2ec42d9_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
osx-64::distributed-2022.2.0-py37h5186d4c_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
osx-64::distributed-2022.2.0-py37hf985489_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
osx-64::distributed-2022.2.0-py38h50d1736_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
osx-64::distributed-2022.2.0-py39h6e9494a_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
osx-arm64::distributed-2021.12.0-py310hbe9552e_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
osx-arm64::distributed-2021.12.0-py38h10201cd_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
osx-arm64::distributed-2021.12.0-py39h2804cbe_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
osx-arm64::distributed-2022.1.0-py310hbe9552e_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
osx-arm64::distributed-2022.1.0-py38h10201cd_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
osx-arm64::distributed-2022.1.0-py39h2804cbe_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
osx-arm64::distributed-2022.1.1-py310hbe9552e_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
osx-arm64::distributed-2022.1.1-py38h10201cd_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
osx-arm64::distributed-2022.1.1-py39h2804cbe_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
osx-arm64::distributed-2022.2.0-py310hbe9552e_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
osx-arm64::distributed-2022.2.0-py38h10201cd_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
osx-arm64::distributed-2022.2.0-py39h2804cbe_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
win-64::distributed-2021.12.0-py310h5588dad_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
win-64::distributed-2021.12.0-py37h03978a9_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
win-64::distributed-2021.12.0-py37h4c0cbd9_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
win-64::distributed-2021.12.0-py38haa244fe_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
win-64::distributed-2021.12.0-py39hcbf5309_0.tar.bz2
-    "dask-core 2021.12.0.*",
+    "dask-core >=2021.12.0,<2021.12.1.0a0",
win-64::distributed-2022.1.0-py310h5588dad_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
win-64::distributed-2022.1.0-py37h03978a9_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
win-64::distributed-2022.1.0-py37h4c0cbd9_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
win-64::distributed-2022.1.0-py38haa244fe_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
win-64::distributed-2022.1.0-py39hcbf5309_0.tar.bz2
-    "dask-core 2022.1.0.*",
+    "dask-core >=2022.1.0,<2022.1.1.0a0",
win-64::distributed-2022.1.1-py310h5588dad_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
win-64::distributed-2022.1.1-py37h03978a9_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
win-64::distributed-2022.1.1-py37h4c0cbd9_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
win-64::distributed-2022.1.1-py38haa244fe_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
win-64::distributed-2022.1.1-py39hcbf5309_0.tar.bz2
-    "dask-core 2022.1.1.*",
+    "dask-core >=2022.1.1,<2022.1.2.0a0",
win-64::distributed-2022.2.0-py310h5588dad_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
win-64::distributed-2022.2.0-py37h03978a9_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
win-64::distributed-2022.2.0-py37h4c0cbd9_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
win-64::distributed-2022.2.0-py38haa244fe_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
win-64::distributed-2022.2.0-py39hcbf5309_0.tar.bz2
-    "dask-core 2022.2.0.*",
+    "dask-core >=2022.2.0,<2022.2.1.0a0",
```

<\details>

File: recipe/gen_patch_json.py [hunk 1]
```diff
@@ -630,6 +630,24 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
             if pversion < v2022_6_1:
                 record['depends'].append('tornado <6.2')
 
+        if record_name in {"distributed", "dask"}:
+            version = pkg_resources.parse_version(record["version"])
+            if (
+                version >= pkg_resources.parse_version("2021.12.0") and
+                version < pkg_resources.parse_version("2022.8.0") or
+                version == pkg_resources.parse_version("2022.8.0") and
+                record["build_number"] < 2
```
Comment (jakirkham) [inline recipe/gen_patch_json.py:639]: Why these constraints?

**Review by jakirkham — COMMENTED** [review]
Thanks Charles! 🙏

Had one question below

File: recipe/gen_patch_json.py (hunk 1 above)
Comment (charlesbluca) [reply recipe/gen_patch_json.py:639]: The oldest available `dask-core` nightlies that we would want to exclude are `2021.12.0a211220`:

https://anaconda.org/dask/dask-core/files?version=2021.12.0a211220

This targets all Dask packages from `2021.12.0` to the older builds of `2022.8.0`. Is there an easier way to target these packages, or do you think we should target more Dask releases?

**Comment by jakirkham** [comment]
Thoughts @conda-forge/core? 🙂

**Pull request merged**